firewall/handlers/handlers_util.go

```diff
@@ -7,8 +7,12 @@ import (
 
 var ErrQueryNotInWhitelist = errors.New("query not in whitelist")
 var ErrQueryInBlacklist = errors.New("query in blacklist")
-var ErrAccessToForbiddenTable = errors.New("query tries to access forbidden table")
-var ErrForbiddenSqlStructure = errors.New("query's structure is forbidden")
+
+var ErrAccessToForbiddenTableBlacklist = errors.New("query tries to access forbidden table | blacklist")
```

Comment:
no need to add `| blacklist` at end of message for ErrAccessToForbiddenTable**Blacklist** error